First_section.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="First_section.css" />
+  </head>
+
+  <body>
+    <div class="container">
+      <section>
+        <header class="toolbar">
+          <h1 id="Logo"><a href="." rel="home">My Logo</a></h1>
```

Comment:
[Optional]

- [ ] Please consider removing the `h1` tag here since in a web page there should only be one `h1` tag which is the main header and it should be the text I previously mentioned 👍 
- [ ] It's better if you put `#` in the `href` attribute, this way clicking on the logo will always take us to the top of the page 💯 
- [ ] Please consider styling the logo according to the Figma template 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/205106906-fb35aeb0-e55d-4212-9733-bd8fa33dee6e.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/205106971-73cc90e3-b857-41cb-b91b-a68defa6ca7c.png)